### ISSUE TITLE
Add History of Code Submissions

### ISSUE THREAD

**Issue by ethanchewy** [ethanchewy/PythonBuddy#48]
As a user, I want to be able to view my previous code submissions.

**Comment by dgmouris** [comment]
I really like this idea, although perhaps this could be a separate application within the repository as you might want to have restricted sandbox for user submitted code and an application that handles all history/login/customization. 

Let me know what you think this is a thought, don't want to overstep bounds. Awesome project!

**Comment by ethanchewy** [comment]
Yup, we would probably build it in: https://github.com/ethanchewy/PythonBuddySandboxed

Feel free to work on it if you want to!

**Comment by ethanchewy** [comment]
Actually, for this repo, assuming that we aren't handling any logins, we can still set up a way for the user to easily check his/her/their code submission history.

**Comment by ethanchewy** [comment]
Edited issue description to reflect this
